### DISCUSSION TITLE
Delete all repositories

### DISCUSSION THREAD

**Discussion by ghost** [community/community#58199] [Repositories] [answered]
How can I delete my repositories.  My ex is using my emails to create accounts to control my phone, accounts, pictures, videos, banking info, you name it.  My GitHub account was also created by him. He has my phone so hacked along with my laptops to where I cannot see what most people can when they go to certain screens.  My options are limited so please keep in mind.  Any help helps.

**Comment by radar-blip** [comment]
You can delete any of your repositories by opening the repository and go to `Settings` at the top of that page.
Then scroll to the bottom and you will see: `Delete Repository`

**Reply by ghost** [reply]
I don't have that option

**Comment by Gustavonn07** [comment, marked as answer]
To delete a repository follow this steps:

1. Go to configs
<hr>

![image](https://github.com/community/community/assets/84361085/7d4cdda8-ce1b-466e-9084-82919e22cfe9)

</hr>

2. Scroll down
<hr>

![image](https://github.com/community/community/assets/84361085/1b6d5313-8a78-4871-a3e0-79e9813a5058)

</hr>

3. Click Delete this repository and write your name/repository
<hr>

![image](https://github.com/community/community/assets/84361085/49b7adda-4a57-48bd-9732-05eccd20e7c1)

</hr>

**Comment by ghost** [comment]
It worked. Thank you all!

**Reply by Rabab-Muhammad** [reply]
its not working
